builtin/logical/ssh/path_sign.go

```diff
@@ -204,6 +261,54 @@ func (b *backend) pathSignCertificate(ctx context.Context, req *logical.Request,
 	return response, nil
 }
 
+// return values:
+//  1. the "valid_after" from specified data, 0 if not specified
+//  2. true if "valid_after" was specified in the data, false otherwise
+//  3. any error
+func (b *backend) calculateValidAfter(data *framework.FieldData) (uint64, bool, error) {
+
+	validAfterRaw, valueWasSet, err := data.GetOkErr("valid_after")
```

Comment:
Not a big deal, but FWIW, it would be possible to just use `GetOK` here and eliminate the type check lines on 273 thru 275. The outer framework will handle and error if it's given input that doesn't parse as the given schema type.